src/injection/container.ts

```diff
@@ -28,6 +28,6 @@ DIContainer.bind<AbstractRouter>("Routers").to(TransactionRouter).inTransientSco
 DIContainer.bind<DataSource>("DataSource").toConstantValue(DataSource.MONGO_DB);
 DIContainer.bind<IClientConfiguration>("Configuration")
     .toConstantValue(new MongoDBConfiguration(`mongodb://${process.env.MONGODB_HOST}:${process.env.MONGODB_PORT}`,
```

Comment:
This is not in sync with the validator.
```suggestion
    .toConstantValue(new MongoDBConfiguration(process.env.DB_CONNECTION_STRING as string));
```